source/configure/experimental-configuration-settings.rst

```diff
@@ -660,6 +660,19 @@ See the `Important Upgrade Notes <https://docs.mattermost.com/upgrade/important-
 | This feature's ``config.json`` setting is ``"PatchPluginsReactDOM": false`` with options ``true`` and ``false``. |
 +------------------------------------------------------------------------------------------------------------------+
 
+Delay channel autocomplete
```

Comment:
@hmhealey - Is this new config setting available to both self-hosted and Cloud deployments? And is it available in TE or does it require Professional or Enterprise?

Reply:
It's available everywhere. There's no license or location requirements on it